<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tarea2</class>
 <widget class="QDialog" name="Tarea2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="BtnAgregar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnInsertar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Insertar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnImprimir">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Imprimir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnEliminar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ListaNArios">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="CmbPadres">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="CmbHijos">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPadre">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Padres:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelHijo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Hijos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelValor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValor">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
